src/Optimizely/Config/ProjectConfigInterface.php

```diff
@@ -51,6 +51,37 @@ public function getBotFiltering();
      */
     public function getRevision();
 
+    /**
+     * @return string String represnting environment key of the datafile.
+     */
+    public function getEnvironmentKey();
+
+    /**
+     * @return string String representing sdkkey of the datafile.
+     */
+    public function getSdkKey();
+
+    /**
+     * @return array List of attributes parsed from the datafile
+     */
+    public function getAttributes();
```

Comment:
following are not a part of ProjectConfigInterface

Reply:
attributes, audiences, events, typedAudiences. 
check in other sdks, how retrieving arrays.